dev/com.ibm.ws.microprofile.health.2.0_fat/test-applications/DelayedHealthCheckApp/src/com/ibm/ws/microprofile/health20/delayed/health/check/app/DelayedServlet.java

```diff
@@ -35,7 +35,7 @@ public DelayedServlet() {
     public void init() {
         System.out.println("Entering init function - Starting Thread.sleep for 30 seconds.");
         try {
-            Thread.sleep(65000); //30 seconds = 30000 ms
+            Thread.sleep(30000); //30 seconds = 30000 ms
```

Comment:
This needs to be 65000, in order for the application Servlet to actually be delayed. Please update the comment and message to reflect that.